lake/index/writer.go

```diff
@@ -101,12 +101,12 @@ func (a *onceError) Load() error {
 }
 
 type indexer struct {
-	err        onceError
-	cutter     *expr.Cutter
-	fgr        zbuf.ReadCloser
-	keyType    zng.Type
-	microindex *microindex.Writer
-	wg         sync.WaitGroup
+	err     onceError
+	cutter  *expr.Cutter
+	fgr     zbuf.ReadCloser
+	keyType zng.Type
+	index   *index.Writer
```

Comment:
Nit: Keep these sorted.
```suggestion
	index   *index.Writer
	keyType zng.Type
```